it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/spanner/SpannerResourceManager.java

```diff
@@ -649,7 +651,15 @@ public Builder useStaticInstance() {
      */
     @SuppressWarnings("nullness")
     public Builder maybeUseStaticInstance() {
```

Comment:
If there is a UT for maybeUseStaticInstance. Please add a test case.

Reply:
There is no test